README.md

```diff
@@ -75,6 +75,14 @@ Get-Help -Name Invoke-SCuBA -Full
 
 ### Parameter Definitions
 
+- **$ConfigFilePath** is an optional configuration file that the tool parses for input parameters when executing ScubaGear. The file can be in YAML or JSON format. A sample configuration file ("aad-config.yaml") is included in the "sample-config-files" sub-folder. The example file includes use of Pascal case convention for varible names. The syntax defines
+  - A global namespace for values to be used across baselines/products (i.e., GlobalVars)
+  - Per product namespace for values related to that specific product (i.e., Aad, SharePoint)
+  - Namespace for each policy item within a product for variables related only to one policy item (i.e., Aad.Policy2_1)
+  - Use of YAML anchors and aliases following DRY (Don't Repeat Yourself) principle for repeated values and sections
+
+  When using the configuration file option all non-default parameters must be specified in the file as the tool ignores other command line options. The file path defaults to the same directory where the script is executed. The file can be used to specify both standard tool parameters as well as custom parameteres used by the Azure Active Directory (AAD) product assessment. See [AAD Conditional Access Policy Exemptions](#aad-conditional-access-policy-exemptions) for more details.
```

Comment:
The tool does not ignore.  It does not allow.  

![image](https://github.com/cisagov/ScubaGear/assets/7025068/d6b4b12b-02c9-401e-925c-1b9131f2d1d7)

Using the PowerShell Get-Help we can see it has three ParameterSets.  

![image](https://github.com/cisagov/ScubaGear/assets/7025068/0b147148-9d63-47ae-9a2f-aa982f6bf81b)

Reply:
Should we mention the <Common Parameters> here? The other examples here didn't mention that parameter set.